src/authV2/azext_authV2/custom.py

```diff
@@ -764,29 +839,51 @@ def add_openid_connect_provider_settings(cmd, resource_group_name, name, provide
         raise CLIError('Usage Error: The following custom OpenID Connect provider has already been '
                        'configured: ' + provider_name + '. Please use az webapp auth oidc update to '
                        'update the provider.')
+
+    final_client_secret_setting_name = client_secret_setting_name
+    if client_secret is not None:
```

Comment:
Do we need to worry about the case where `client_secret` is None? If I'm reading the code correctly (the indenting might be throwing me off), it seems like we'll try to set the `clientSecretSettingName` regardless of whether a client secret was actually provided.

Reply:
So per the code, if the client secret is not None, we set the app setting and set the clientSecretSettingName to whatever name we just gave that app setting we set. If client secret is none, we set clientSecretSettingName to the value of whatever is passed in to --client-secret-setting-name. If it's none, we leave it none, if it's not none, we set it to the specified value. Does that make sense, or do you recommend changing that?

Reply:
Makes sense. I forgot that when `client_secret` is none, it's likely because `client_secret_setting_name` was passed instead. Thanks!